content/en/docs/concepts/traffic-management/index.md

```diff
@@ -908,60 +532,34 @@ spec:
-
-- Define
-   [retry](/docs/concepts/traffic-management/#timeouts-and-retries),
-   [timeout](/docs/concepts/traffic-management/#timeouts-and-retries),
-   and [fault injection](/docs/concepts/traffic-management/#fault-injection)
-   policies for external destinations.
-
-- Add a service running in a Virtual Machine (VM) to the mesh to [expand your mesh](/docs/examples/mesh-expansion/).
+## Service entries {#service-entries}
 
-- Logically add services from a different cluster to the mesh to configure a
-  [multicluster Istio mesh](/docs/setup/install/multicluster/gateways/#configure-the-example-services)
-  on Kubernetes.
+You use a
+[service entry](/docs/reference/config/networking/v1alpha3/service-entry) to add
+an entry to the service registry that Istio maintains internally. After you add
+the service entry, the Envoy proxies can send traffic to the service as if it
+was a service in your mesh. Configuring service entries allows you to manage
+traffic for services running outside of the mesh, including the following tasks:
 
-You don’t need to add a service entry for every external service that you
-want your mesh services to use. By default, Istio configures the Envoy proxies
-to passthrough requests to unknown services, although you can't use Istio features
-to control the traffic to destinations that are not registered in the mesh.
+-   Redirect and forward traffic for external destinations, such as APIs
+    consumed from the web, or traffic to services in legacy infrastructure.
+-   Define [retry](#retries), [timeout](#timeouts), and
+    [fault injection](#fault-injection) policies for external destinations.
+-   Add a service running in a Virtual Machine (VM) to the mesh to
+    [expand your mesh](/docs/examples/mesh-expansion/single-network/#running-services-on-a-mesh-expansion-machine).
+-   Logically add services from a different cluster to the mesh to configure a
+    [multicluster Istio mesh](/docs/setup/install/multicluster/gateways/#configure-the-example-services)
+    on Kubernetes.
 
-You can use service entries to perform the following configurations:
+You don’t need to add a service entry for every external service that you want
+your mesh services to use. By default, Istio configures the Envoy proxies to
+passthrough requests to unknown services. However, you can’t use Istio features
+to control the traffic to destinations that aren't registered in the mesh.
 
-- Access secure external services over plain text ports,
-  to configure Envoy to perform {{< gloss >}}TLS Origination{{</ gloss >}}.
-- Ensure, together with an egress gateway, that all external services are
-  accessed through a single exit point.
-
-Refer to the [Egress task topic](/docs/tasks/traffic-management/egress/) for details.
-
-## Add an external dependency securely
+### Service entry example {#service-entry-example}
 
 The following example mesh-external service entry adds the `ext-resource`
```

Comment:
```suggestion
The following example mesh-external service entry adds the `ext-resource.com`
```

Reply:
maybe a better name would be `ext-svc.example.com`?

Reply:
Agreed. Done.